Directory.Build.props

```diff
@@ -34,11 +34,6 @@
     <DashboardPublishedArtifactsOutputDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'DashboardArtifacts', '$(Configuration)'))</DashboardPublishedArtifactsOutputDir>
     <WorkloadsPackageSource>$(ArtifactsShippingPackagesDir)</WorkloadsPackageSource>
 
-    <!-- Issue: https://github.com/dotnet/aspire/issues/8488 -->
-    <!-- xUnit2031: Do not use Where clause with Assert.Single -->
-    <!-- xUnit1051: Calls to methods which accept CancellationToken should use TestContext.Current.CancellationToken to allow test cancellation to be more responsive. -->
-    <!-- TODO: Re-enable and remove this. -->
-    <NoWarn>$(NoWarn);xUnit2031;xUnit1051</NoWarn>
```

Comment:
Is xUnit2031 being addressed here as well?

Reply:
No.
Just moved to a different file.
Moved back xUnit2031